src/overlay/views/Overlay.js

```diff
@@ -21,9 +21,10 @@ var
 	$window = $(win),
 	template = _.template(Template),
 	gettext = win.gettext,
+	previousStyle,
```

Comment:
This variable is in a global scope of the overlay class. Any instance of overlay will have access to this value. So this will lead to unexpected results when reset the overflow value.

You're already using the `this._previousOverflow` proprerty. Why an other variable?